packages/http-json-body-parser/index.d.ts

```diff
@@ -1,19 +1,21 @@
 import middy from '@middy/core'
-import { APIGatewayEvent } from 'aws-lambda'
+import { APIGatewayEvent, APIGatewayProxyEventV2 } from 'aws-lambda'
 import { JsonValue } from 'type-fest'
 
 interface Options {
   reviver?: (key: string, value: any) => any
   disableContentTypeError?: boolean
 }
 
-export type Event = Omit<APIGatewayEvent, 'body'> & {
+type VersionedApiGatewayEvent = APIGatewayEvent | APIGatewayProxyEventV2
+
+export type Event<APIGatewayEventType extends VersionedApiGatewayEvent = VersionedApiGatewayEvent> = Omit<APIGatewayEventType, 'body'> & {
```

Comment:
BTW, I think that at some point we should rename this to ParsedEvent,
not super crucial, but it conveys its meaning more precisely.

@lbertenasco wdyt?

Reply:
I like the suggestion, can we start by doing a type alis and deprecating the old one? Thoughts @willfarrell ?